package.json

```diff
@@ -1,67 +1,66 @@
 {
-  "name": "inversify-express-utils",
-  "version": "6.4.7",
   "author": "Cody Simms",
-  "description": "Inversify wrapper for the express framework",
-  "license": "MIT",
-  "main": "lib/index.js",
-  "module": "es/index.js",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/inversify/inversify-express-utils.git"
-  },
-  "scripts": {
-    "build": "npm run build:lib && npm run build:es && npm run build:es6",
-    "build:lib": "tsc -p src/tsconfig.json",
-    "build:es": "tsc -p src/tsconfig-es.json",
-    "build:es6": "tsc -p src/tsconfig-es6.json",
-    "clean": "rm -r es es6 lib",
-    "test": "jest",
-    "test:coverage": "jest --coverage",
-    "test:watch": "jest --watch",
-    "pretest": "eslint ./",
-    "eslint:fix": "eslint --fix ./",
-    "publish-please": "publish-please",
-    "prepublish": "npm run build && publish-please guard",
-    "update": "updates --update --minor && (git diff-files --quiet package.json || (rimraf package-lock.json node_modules && npm install))",
-    "postupdate": "git diff-files --quiet package-lock.json || npm test"
-  },
   "bugs": {
     "url": "https://github.com/inversify/inversify-express-utils/issues"
   },
-  "homepage": "https://github.com/inversify/inversify-express-utils#readme",
-  "jsnext:main": "es/index.js",
-  "keywords": [
-    "InversifyJS",
-    "express"
-  ],
   "dependencies": {
-    "express": "4.21.1",
-    "http-status-codes": "2.3.0",
-    "inversify": "^6.0.3"
+    "http-status-codes": "2.3.0"
   },
+  "description": "Some utilities for the development of express applications with Inversify",
   "devDependencies": {
     "@types/async": "3.2.24",
     "@types/cookie-parser": "1.4.7",
-    "@types/express": "4.17.21",
+    "@types/express": "^4.17.21",
     "@types/jest": "29.5.14",
-    "@types/node": "22.8.0",
```

Comment:
Why not update node version?

Reply:
I prefer to keep the types associated to the current LTS